recipes/tardis/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "tardis-sn" %}
+{% set version = "3.0.a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: true  # [not win]
```

Sorry I missed this before, it currently skips all platforms except windows. I think you want this instead:
```suggestion
  skip: true  # [win]
```